dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/security/impl/ldap/LdapService.java

```diff
@@ -67,6 +79,12 @@ public class LdapService {
     @Value("${security.authentication.ldap.user.email-attribute:#{null}}")
     private String ldapEmailAttribute;
 
+    @Value("${security.authentication.ldap.user.admin-attribute:#{null}}")
+    private String ldapAdminIdentifingAttribute;
+
+    @Value("${security.authentication.ldap.user.admin-value:#{null}}")
+    private String ldapAdminIdentifingValue;
```

Comment:
You should add this to config and docs.

Reply:
Added config to documentation. Cannot find any LDAP document page on how to use the attributes?

Reply:
not sure what you mean by config?

Reply:
You can take a look at https://github.com/apache/dolphinscheduler/blob/4f6e7159b8b635b615de9fdc4be7932e324e4134/dolphinscheduler-api/src/main/resources/application.yaml#L169-L191